community/php7/APKBUILD

```diff
@@ -137,231 +143,244 @@ _build() {
 	_build --disable-phpdbg \
 		--disable-cgi \
 		--disable-cli \
 		--with-apxs2 \
+		--with-recode=shared \
 		|| return 1
-	mv libs/libphp7.so sapi/apache2handler/mod_php7.so
-	# cgi,cli,fpm,embed,litespeed
+	mv libs/lib$pkgname.so sapi/apache2handler/
+	# build all other + imap extension
 	_build --disable-phpdbg \
 		--enable-fpm \
 		--enable-embed \
 		--with-litespeed \
+		--with-imap=shared --with-imap-ssl \
 		|| return 1
 }
 
 package() {
 	cd "$builddir"
-
 	make -j1 INSTALL_ROOT="$pkgdir" install || return 1
 
-	install -Dm644 php.ini-production "$pkgdir"/etc/$pkgname/php.ini || return 1
+	rm -fr "$pkgdir"/.[[:alpha:]]* || return 1
 
-	local file; for file in pear peardev pecl; do
-		sed -i -e "s~/usr/bin/php~/usr/bin/php7~g" \
-			-e "s~PHP=php~PHP=php7~" \
-			"$pkgdir"/usr/bin/$file || return 1
-	done
-	paxmark -m "$pkgdir"/usr/bin/php7
-
-	find "$pkgdir" -name '.*' | xargs rm -rf || return 1
-	rmdir "$pkgdir"/var/run
+	ln -sf php$_suffix "$pkgdir"/usr/bin/php
 }
 
 dev() {
 	default_dev
-	depends="$depends $pkgname $pkgname-pear"
 
 	mkdir -p "$subpkgdir"/usr/bin \
 		"$subpkgdir"/usr/lib/$pkgname
 
-	mv "$pkgdir"/usr/bin/php-config7 "$pkgdir"/usr/bin/peardev \
-		"$subpkgdir"/usr/bin/ || return 1
+	local file
+	for file in php-config phpize; do
+		mv "$pkgdir"/usr/bin/$file$_suffix \
+			"$subpkgdir"/usr/bin/ || return 1
+		ln -sf $file$_suffix "$subpkgdir"/usr/bin/$file
+	done
+
 	mv "$pkgdir"/usr/lib/$pkgname/build \
-		"$subpkgdir"/usr/lib/$pkgname/
+		"$subpkgdir"/usr/lib/$pkgname/ || return 1
 }
 
 doc() {
 	default_doc
+	install_if="docs $pkgname-common=$pkgver-r$pkgrel"
 
-	mkdir -p "$subpkgdir"/usr/share/doc/$pkgname
-
+	cd "$builddir"
 	local file
-	for file in CODING_STANDARDS CREDITS EXTENSIONS INSTALL LICENSE NEWS README* UPGRADING*; do
-		cp "$builddir"/$file "$subpkgdir"/usr/share/doc/$pkgname/ || return 1
+	for file in $(ls CODING_STANDARDS CREDITS EXTENSIONS INSTALL LICENSE NEWS README* UPGRADING*); do
+		install -Dm644 $file "$subpkgdir"/usr/share/doc/$pkgname/$file || return 1
 	done
 }
 
 apache2() {
-	pkgdesc="PHP Module for Apache2"
+	pkgdesc="$pkgdesc (apache2 module)"
 	depends="$pkgname-common apache2"
 
-	install -D -m755 "$builddir"/sapi/apache2handler/mod_php7.so \
-		"$subpkgdir"/usr/lib/apache2/mod_php7.so || return 1
+	install -D -m755 "$builddir"/sapi/apache2handler/lib$pkgname.so \
+		"$subpkgdir"/usr/lib/apache2/mod_$pkgname.so || return 1
 
-	install -D -m644 "$srcdir"/php7-module.conf \
-		"$subpkgdir"/etc/apache2/conf.d/php7-module.conf
+	install -D -m644 "$srcdir"/$pkgname-module.conf \
+		"$subpkgdir"/etc/apache2/conf.d/$pkgname-module.conf
 }
 
 phpdbg() {
-	pkgdesc="Interactive PHP debugger"
+	provides="php-phpdbg"
+	pkgdesc="$pkgdesc (interactive debugger)"
 	depends="$pkgname-common"
 
 	install -Dm755 "$builddir"/sapi/phpdbg/phpdbg \
-		 "$subpkgdir"/usr/bin/phpdbg7
+		 "$subpkgdir"/usr/bin/phpdbg$_suffix || return 1
+	ln -sf phpdbg$_suffix "$subpkgdir"/usr/bin/phpdbg
```

Comment:
@jirutka 
cli, cgi, phpdbg and dev-tools are in conflict with php5 ones for now.